lib/backend.js

```diff
@@ -353,20 +350,28 @@ Backend.prototype.getOpsBulk = function(agent, index, fromMap, toMap, callback)
 Backend.prototype.fetch = function(agent, index, id, callback) {
   var start = Date.now();
   var projection = this.projections[index];
-  var collection = (projection) ? projection.target : index;
+  var dbCollection = (projection) ? projection.target : index;
```

Comment:
Let's leave this as `collection` for this PR. we could consider changing it, but we'd want to be consistent everywhere